apps/web/src/app/(dashboard)/documents/delete-document-dialog.tsx

```diff
@@ -39,10 +41,16 @@ export const DeleteDocumentDialog = ({
   const { mutateAsync: deleteDocument, isLoading } = trpcReact.document.deleteDocument.useMutation({
     onSuccess: () => {
       router.refresh();
+      const deletedFileToastDescription = (
+        <p>
+          Your document <span className="text-documenso">{documentTitle}</span> has been
```

Comment:
We should probably avoid using green text on a white background, according to the WCAG compliance results. What do you think?

![CleanShot 2023-12-20 at 17 37 12](https://github.com/documenso/documenso/assets/25515812/825f7a69-4070-4b90-ad61-abcce8443ce9)

Reply:
Yes Yes, I will update it to just be in quotes instead of the green text 👀